src/layout/section/kangarooSection/kangarooSection.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import cc from 'classcat'
+
+export interface KangarooSectionProps {
+  readonly className?: Classcat.Class
+  readonly role?: string
+  readonly children: JSX.Element
```

Comment:
I got a feedback on a previous PR that we should use `React.ReactNode` instead cc @grosbouddha

Reply:
React.ReactNode should work with string, JSX and array of JSX and is the most flexible.

Reply:
Ok, i'll change that